models/nc-news-models.js

```diff
@@ -125,6 +125,18 @@ function removeComment(comment_id) {
     });
 }
 
+function fetchUsers() {
+  return db.query(`SELECT * FROM users;`).then(({ rows }) => {
+    if (!rows[0]) {
+      return Promise.reject({
+        status: 404,
+        msg: "There are no users yet",
+      });
```

Comment:
no way to test this works.  but I can see why you've put it in (even though it's not strictly necessary following the TDD principles )